### ISSUE TITLE
Improve structure of tests

### ISSUE THREAD

**Issue by wotzlaff** [romgrk/node-gtk#228]
As I always have problems with writing a new test, we could:
- [ ] add a template for new test files
- [ ] use consistent importing of `__common__.js` to get rid of the `common.`-prefix
- [ ] ensure that all test follow a common structure and style
- [ ] merge test files when possible (those with common prefix and `__` in the filename)

**Comment by romgrk** [comment]
Yes, that would be great. The `__` thing was because I had a lot of segfaults back then and I wanted to know which things did a segfault :laughing: But now having many test files slows down things more than it help.
